docs/reference/aggregation.md

```diff
@@ -0,0 +1,488 @@
+---
+title: Aggregation
+sidebar_position: 11
+---
+
+An aggregation summarizes your data as statistics on buckets or metrics.
+
+Aggregations can provide answer to questions like:
+
+- What is the average price of all sold articles?
+- How many errors with status code 500 do we have per day?
+- What is the average listing price of cars grouped by color?
+
+There are two categories: [Metrics](#metric) and [Buckets](#bucket-aggregations).
+
+#### Prerequisite
+
+To be able to use aggregations on a field, the field needs have a fast field index created. A fast field index is a columnar storage, 
+where documents values are extracted and stored to.
+
+Example to create a fast field on text for term aggregations.
```

Comment:
Maybe mention the fact that the text field can be a fast field only if there is not tokenization

Reply:
It's mentioned later, text field supports also tokenization with fast field but is not recommended